tests/system/environmentLink_test.go

```diff
@@ -20,7 +20,6 @@ func TestEnvironmentLink(t *testing.T) {
 	defer s.Terraform.Destroy()
 
 	publicEnvironmentID := s.Terraform.Output("public_environment_id")
-	privateEnvironmentID := s.Terraform.Output("private_environment_id")
```

Comment:
why is this gone?

Reply:
Because of how links work now (that they aren't bi-directional), I had attempted to modify main.tf for this system test case to create one link from public to private. What ended up happening was I only needed to inspect the public environment's links, thus this variable was not needed. The terraform files are now wrong after my reversion because you have an environment link block that looks like this:

```
resource "layer0_environment_link" "public_private" {
  source = "${layer0_environment.el_public.id}"
  dest = "${layer0_environment.sl_private.id}"
}
```

But this structure is wrong.

I'm just going to reverse this removal for now but fixing this will require fixing the terraform files more fully.

Reply:
Actually, I see more clearly why this is gone now. We need to call `s.Layer0.UpdateEnvironmentLink(publicEnvironmentID, links)` whereas before this was `s.Layer0.DeleteLink(publicEnvironmentID, privateEnvironmentID)`

So unless I change how this test is written to make two unidirectional links, this variable needs to stay out.

Reply:
sounds good, expect `s.Layer0.UpdateEnvironmentLink(publicEnvironmentID, links)` is not a thing that exists nor should exist. This is handled by the `UpateEnvironment` api call.

Reply:
Yes, I intended to say `UpdateEnvironment` not `UpdateEnvironmentLink`

Reply:
The code already reflects the usage of `UpdateEnvironment`